spring/boot2-webflux-autoconfigure/src/test/java/com/linecorp/armeria/spring/web/reactive/ArmeriaReactiveWebServerFactoryTest.java

```diff
@@ -80,10 +85,18 @@ private WebClient httpClient(WebServer server) {
 
     @Test
     void shouldRunOnSpecifiedPort() {
-        final ArmeriaReactiveWebServerFactory factory = factory();
-        final int port = SocketUtils.findAvailableTcpPort();
-        factory.setPort(port);
-        runEchoServer(factory, server -> assertThat(server.getPort()).isEqualTo(port));
+        for (int i = 0; i < 3; i++) {
+            final ArmeriaReactiveWebServerFactory factory = factory(new DefaultListableBeanFactory());
```

Comment:
Could you leave a comment on why we try this three times?

Reply:
Yap!